GeosCore/diagnostics_mod.F90

```diff
@@ -133,6 +134,20 @@ SUBROUTINE Set_Diagnostics_EndofTimestep( Input_Opt,  State_Chm,           &
        RETURN
     ENDIF
 
+    !-----------------------------------------------------------------------
+    ! Set species concentration for diagnostics in units of
+    ! molec/cm3 (hplin, 11/21/21)
+    !-----------------------------------------------------------------------
+    CALL Set_SpcConc_Diags_MND  ( Input_Opt,  State_Chm, State_Diag,         &
+                                  State_Grid, State_Met, RC                 )
+
+    ! Trap potential errors
+    IF ( RC /= GC_SUCCESS ) THEN
+       ErrMsg = 'Error encountered setting species concentration diagnostic 2'
```

Comment:
Suggest renaming error message to "Error encountered setting SpeciesConcMND diagnostic"

Reply:
Now fixed in [1666335b](https://github.com/geoschem/geos-chem/commit/1666335bda77e786388cc19f214b4d203cbc3117).